### DISCUSSION TITLE
Welcome to birthdayApp Discussions!

### DISCUSSION THREAD

**Discussion by 40PlusBaddies** [40PlusBaddies/birthdayApp#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marianaWest** [comment]
Hi everybody! I'm still not that deep into the back end so I'll try to contribute more with front end stuff. I started creating a Figma just to understand what input fields we would like to have, please don't pay attention to the design (or lack of it). Also, I'm obviously not an English native speaker so always feel free to correct my spelling/grammar. 
On that note, I think it would be nice to make sure that the user is allowed to choose between different date formats. F.ex., we write the day/month, not month/day like Americans. 
https://www.figma.com/file/efWfl4DtuFKpIrkmJgD7Tv/birthdayApp?node-id=0%3A1
let's get this started!

**Comment by freddie1717** [comment]
I’m terrible at the front end so this looks it will be a good fit! I’m not exactly an expert at the backend but having just completed the project with my other group, I feel like I understand it a lot more. I would really like to have a group call to just talk through the project, I find it easier than typing everything out sometimes!

**Comment by borojetski** [comment]
Hi everyone! Working things out live sounds like a good idea. I’m guessing the first few steps will be forking the starter code and maybe creating different branches? I’m really looking forward to this!
